clubs/views.py

```diff
@@ -225,6 +227,25 @@ class MajorViewSet(viewsets.ModelViewSet):
     queryset = Major.objects.all()
 
 
+class YearViewSet(viewsets.ModelViewSet):
+    """
+    list:
+    Retrieve a list of all of the graduation years (ex: Freshman, Sophomore, Junior, Senior).
```

Comment:
should it be text and not numbers like 2022?

Reply:
This is actually intentionally text. The reason for this is that we want club members to specify "we accept Freshman, Sophomore, and Juniors" instead of "we accept class of 2023, 2022, and 2021". We don't want the club to have to update the graduation years they accept every year, since this information is likely to fall out of date if they don't notice.

We could add an `offset` field from the current year to make calculations easier on the client side, but we would need to consider the edge cases related with the current time of year.

If you look at the user profile field, you can see that the graduation year is indeed a number. This is because this field probably will not change from year to year.

Reply:
ahh gotcha makes sense